src/Manifest.cc

```diff
@@ -415,26 +415,38 @@ validateDepName(const StringRef name) {
     throw PoacError("dependency name must start with an alphanumeric character"
     );
   }
-  // Trailing `-`, `_`, and `/` are not allowed.
+  // Trailing `-`, `_`, `.`, and `/` are not allowed.
```

Comment:
Why is the order not consistent with the below comment? Why not this?

```suggestion
  // Trailing `-`, `_`, `/`, and `.` are not allowed.
```